LICENSE

```diff
@@ -1,23 +1,21 @@
-                    GNU GENERAL PUBLIC LICENSE
-                       Version 3, 29 June 2007
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
?

Reply:
I used another module as a template , probably used the wrong one should have used Skelton mod , I’ll fix it 👍

Reply:
I don't know much about licenses but it should stay as original, should revert the license files (or lines)

Reply:
there i grabbed the original for this module from ac repo